server/src/server.ts

```diff
@@ -82,6 +84,29 @@ export class Server {
     return sock;
   }
 
+  private updateTemplate(template: Template): Template {
+    template.hash = crypto.createHash("sha256")
+      .update(JSON.stringify(template))
+      .digest("hex");
+
+    const cb0 = "03";                                       // push
+    const cb1 = this.toLittleEndianNumber(template.height); // height
```

Comment:
Is there something that makes sure that this field will be 3 bytes long, [as required by BIP34 (version 2 blocks)](https://github.com/bitcoin/bips/blob/master/bip-0034.mediawiki#specification)? Maybe I'm just missing it, but I don't see anything in `toLittleEndianNumber` or around that would do that.